### PR TITLE
Top level readme

### PR DESCRIPTION
**Pull request by jeberl**
summary: Edit the top level read me to match the new layout and current contents of the top level directory

test plan: Ensure all links to other files in the repo are still correct

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,32 +2,19 @@
 
 ThreatExchange is a set of RESTful APIs on the Facebook Platform for querying, publishing, and sharing security threat information. It's a lightweight way for exchanging details on malware, phishing pages, and other threats with either specific members of the community or the ThreatExchange community at large.
```
Comment (Dcallies) [inline README.md:3]: Do we need to add a "This repository contains a collection of sub-projects that are affiliated with ThreatExchange, or support data formats shared on ThreatExchange" type statement somewhere here, to into into subprojects?

File: README.md [hunk 2]
```diff
@@ -2,32 +2,19 @@
...
 
-This repository contains example code for using the API.
+The `api-reference-examples` folder contains example implementations in various languages for using the API. These implementations are at various stages of completeness and may not all implement every endpoint availible on the ThreatExchange API. For full details on the ThreatExchange API and UI, data formats, and best practices are available in the [ThreatExchange docs](https://developers.facebook.com/docs/threat-exchange/).
```
Comment (Dcallies) [inline README.md:5]: Put an H2 header above this section

Comment (Dcallies) [reply README.md:5]: +1 to referencing public docs

File: README.md (hunk 1 above)
Comment (Dcallies) [inline README.md:3]: We should add a link to the public documentation very early on in the document.

https://developers.facebook.com/docs/threat-exchange

File: README.md [hunk 3]
```diff
@@ -2,32 +2,19 @@
...
-
-For tag-driven workloads, supporting either bulk download or incremental updates, our currently recommended best practice is a [Java reference design](https://github.com/facebook/ThreatExchange/blob/master/hashing/te-tag-query-java/README.md).
-
-We also offer scripts in the `pytx/scripts` folder. For example, to get threat
-descriptors uploaded to ThreatExchange run the following command:
-
-```
-$ python pytx/scripts/get_data.py -o threat_descriptor
-```
+You can also explore the dataset using the hosted [ThreatExchange UI](https://developers.facebook.com/docs/threat-exchange/ui)
```
Comment (Dcallies) [inline README.md:17]: Would move this into the API examples section or a specific note in the public documentation

File: README.md [hunk 4]
```diff
@@ -2,32 +2,19 @@
...
 
 ## Blog
```
Comment (Dcallies) [inline README.md:20]: Empty?

Comment (jeberl) [reply README.md]: Removed

File: README.md (hunk 3 above)
Comment (jeberl) [reply README.md]: Moved to top level summary

File: README.md (hunk 1 above)
Comment (jeberl) [reply README.md:3]: Done

File: README.md (hunk 2 above)
Comment (jeberl) [reply README.md:9]: Done

File: README.md (hunk 1 above)
Comment (jeberl) [reply README.md:3]: Added H2 header right below. I think makes it clear

**Pull request merged**